flutter/shell/platform/tizen/tizen_renderer_nui_gl.cc

```diff
@@ -0,0 +1,71 @@
+// Copyright 2025 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/tizen/tizen_renderer_nui_gl.h"
+#include "flutter/shell/platform/tizen/tizen_renderer_egl.h"
+#include "tizen_renderer.h"
```

Comment:
```suggestion
#include "flutter/shell/platform/tizen/tizen_renderer.h"
```
How about changing like this?